test/steal-noop.js

```diff
@@ -0,0 +1,3 @@
+exports.translate = function(load) {
```

Comment:
Since you're just using normalize() I don't think this file is even needed. It's not loaded until `locate` occurs, which doesn't in your test.